main.scss

```diff
@@ -0,0 +1,103 @@
+@import 'o-colors/main';
+@import 'src/scss/variables';
+@import 'src/scss/functions';
+@import 'src/scss/mixins';
+
+/// Output All oMeter Features
+@mixin oMeter() {
+	@include oColors;
+
+	:root {
+		--o-meter-width: 100%; // we need width to synch .o-meter-container with .o-meter-value to display the value label in the right position
+		--o-meter-height: 1em;
+	}
+
+	.o-meter {
+		display: block;
+		width: var(--o-meter-width);
+		height: var(--o-meter-height);
+		--o-meter-background-color: #{oColorsByName('slate-white-15')};
+		--o-meter-optimum-color: #{oColorsByName('jade')};
+		--o-meter-sub-optimum-color: #ff8833; // 'mandarin' color used as hex because isn't an internal brand color
+		--o-meter-sub-sub-optimum-color: #{oColorsByName('crimson')}; // bad
+
+		/* Background in Firefox */
+		background: var(--o-meter-background-color);
+	}
+
+	.o-meter-container {
+		width: var(--o-meter-width);
+		position: relative;
+		.o-meter {
+			width: 100%;
+		}
+	}
+
+	.o-meter-value {
+		position: relative;
+		padding: 0 5px;
```

Comment:
It would be great to use [o-spacing](https://registry.origami.ft.com/components/o-spacing@2.0.0/readme?brand=master) to set the padding. It helps us keep our margins and paddings a little more consistent.

Reply:
good idea!